core/frontend/src/components/app/PowerMenu.vue

```diff
@@ -63,7 +63,8 @@
             </v-col>
             <v-col>
               <v-btn
-                v-tooltip="'Restarts the core container alone, should be enough in most cases'"
+                v-tooltip="'Restarts the core 
+                alone, should be enough in most cases'"
```

Comment:
Is this line break right?

Reply:
our CI didn't like it...

Reply:
haha, I was wondering why the CI broke, no, I didn't intend to add that in - I'll fix it.